### ISSUE TITLE
Alarm Clock build - feedback from anonymous contributor by email

### ISSUE THREAD

**Issue by cefn** [ShrimpingIt/website_text#6]
This applies to the [Alarm Clock Build webpage](http://start.shrimping.it/project/alarmclock/build.html) generated from markdown source [here](https://github.com/ShrimpingIt/website_text/blob/master/src/content/project/alarmclock/build.md) 

AlarmClock circuit:
- top tabs don’t work (mac - Chrome), but left hand menu is fine
- tab 5: pin 4 - is it a ‘rail’ (cf. breadboard rail from tab 4)? pins 5/6 - do the acronyms SCL/SDA need expanding?
- tab 6: you switch from pin numbers to breadboard numbers, e.g. could you say ‘pin 8 of the DS1307’ and ‘pin 4 of the DS1307’? Breadboard numbers are obviously totally dependent on where the chip is placed ... to me, the pin numbers is educational, the breadboard numbers are not!
- tab 7: again, maybe give pin numbers instead of breadboard numbers?
Colour of wires: obviously red and green are used by convention (do you say this earlier?) but what about purple and yellow - any convention here?
- tab 8: maybe repeat that resistors can go either way round?
- tab 9: crystal polarity?
- tab 10, 12, 13: breadboard numbers again?
- tab 11: what do the 2 pins represent?
- tab 14: last para is slightly confusing I think? Removing the battery will stop it shorting by definition?! So is the coin trick since it is also necessary to ground pin 3 for the circuit to work?
- tab 15: maybe need to explain what the coin battery was powering cf. the 3xAAA batteries?
Maybe as per https://github.com/ShrimpingIt/website_text/issues/5 , what would this look like as a classic circuit diagram?

**Comment by benjaminwohl** [comment]
another comment on the alarmaclock build- that slide 19 doesn't end very well. it doesn't suggest what sketchs to try next etc. (like was done at the half way point). it say now you can use the buttons but it doesn't really say how.

**Comment by cefn** [comment]
Yeah, the buttons need some example sketches. I wonder if now they are there, we could merge the 'demo' functionality with a time-setting functionality by wiring them up as 'fast forward' and 'fast reverse' for the Clock time, meaning that any clock behaviours can be easily dry-run, and also that you can easily set the hour. That would demonstrate how to use them (there is no demo right now - it's been offered up to SCC as a design challenge to use the buttons)

**Comment by benjaminwohl** [comment]
that sounds good…

I sort of got to the end and though…um, now what?… 
also the time seems to be wrong, which makes no sense to me but there you go. (i though it would be pulling form the system time from he computer?)

**Comment by cefn** [comment]
Thanks for your comment, Ben. I think this is a different issue isn't it? Could you use the new issue button and report it as a single thing which can be dealt with?
